### DISCUSSION TITLE
Can you publish a example use gpu+cpu+nvme to train big model ？

### DISCUSSION THREAD

**Discussion by dy2009** [hpcaitech/ColossalAI#1593] [Community | General]
Can you publish a example use gpu+cpu+nvme to train big model  use 1 gpu ?
I have tryed ColossalAI-Examples , but no success.
also RuntimeError: CUDA out of memory.

**Comment by ver217** [comment]
For NVME offload, all you need to do is to replace the optimizer with HybridAdam. For more details, see https://colossalai.readthedocs.io/en/latest/colossalai/colossalai.nn.optimizer.hybrid_adam.html#colossalai.nn.optimizer.hybrid_adam.HybridAdam
